chapters/logging.rst

```diff
@@ -0,0 +1,244 @@
+   * - FluentHandler
+     - Given the distributed nature of Jina's Peas and Pods, Jina offers a flexible solution that lets the user configure how and where the logs are forwarded.
+     - Need to set ``hostname``, ``process`` ``levelname``
+     - FluentD then will have its own configuration to forward the messages according to its own syntax
+
+
+Logging configuration in Action
+--------------------------------------
+
+Jina logging can be configured in YAML file.
+
+.. highlight:: python
+.. code-block:: python
+
+        from jina.flow import Flow
+        from jina import Document
+
+        f = Flow(log_config='logging_cfg.yml').add().add()
+        # If you want to set the configuration for a certain Pod
+        #f = Flow().add(log_config='logging_cfg.yml').add()
+
+In YAML file you can customized the logger, choose the handler you need to monitor and save logs. ``logging_cfg.yml`` can be configured as follows:
+
+.. highlight:: yaml
+.. code-block:: yaml
+
+    handlers:  # enabled handlers, order does not matter
+      - StreamHandler
+      - SysLogHandler
+      - FluentHandler
+    level: INFO  # set verbose level
+    configs:
+      FileHandler:
+        format: '%(asctime)s:{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        output: 'jina-{uptime}.log'
+        formatter: JsonFormatter
+      StreamHandler:
+        format: '{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        formatter: ColorFormatter
+      SysLogHandler:
+        ident: # this will be prepend to all messages
+        format: '{name:>15}@%(process)2d[%(levelname).1s]:%(message)s'
+        host: # when not given then record it locally, /dev/log on linux /var/run/syslog on mac
+        port: # when not given then record it locally,  /dev/log on linux /var/run/syslog on mac
+        formatter: PlainFormatter
+      FluentHandler:
+        # this configuration describes where is the fluentD daemon running and waiting for logs to be emitted.
+        # FluentD then will have its own configuration to forward the messages according to its own syntax
+        # prefix will help fluentD filter data. This will be prepended for FluentD to easily filter incoming messages
+        tag: jina
+        host: 0.0.0.0
+        port: 24224
+        format:
+          host: '%(hostname)s'
+          process: '%(process)s'
+          type: '%(levelname)s'
+
+
+If you want to hide the logs of a certain Pod, you can set ``quiet=True`` in flow like:
+
+.. highlight:: python
+.. code-block:: python
+
+        f = Flow().add(quiet=True).add()
+        with f:
+            f.index(Document())
+
+Or in YAML flow configuration:
+
+.. highlight:: yaml
+.. code-block:: yaml
+
+    !Flow
+    pods:
+      - uses: Pod1.yml
+        quiet: true
+      - uses: Pod2.yml
+
+
+What's more
+-------------
+
+FluentD
+^^^^^^^^^
+Fluentd is an open source data collector for unified logging layer [https://www.fluentd.org/](https://www.fluentd.org/).
```

Comment:
I remember rst has a different way to write external hyperlinks. 

Reference:
 https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#internal-and-external-links

```suggestion
`Fluentd <https://github.com/fluent/fluentd>`_ is an open source data collector for unified logging layer.
```

Reply:
Yes. This section is written by markdown before. I just missed this link. Will update.